tests/unit/backend/diagnostics/test_rotation.py

```diff
@@ -1,13 +1,21 @@
-#!/usr/bin/env python3
+from functools import cache
 import numpy as np
 import pytest
 import tomopy
 from imars3d.backend.diagnostics.rotation import find_rotation_center
 
+# all tests share a consistent set of omeaga angles
+# these are in radians
+OMEGAS = np.linspace(0, np.pi * 2, 181)
 
+# decorator creates a dict of previous parameter/return pairs
+# the cache does not empty
+# each call takes ~1s
+
+
+@cache
```

Comment:
This is a really nice way to reduce computation in unit testing.
Could you add a brief instruction in the developer doc for future developers?